src/components/tab/FileTabHeader.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+  <q-tab
+    :name="file.id"
+    no-caps
+    :ripple="false"
+    :class="['tab-item', { 'active': isActive }]"
+    data-cy="file-tabs-container"
+  >
+    <div
```

Comment:
is this div necessary ?

Reply:
I can remove this div, but I'll have to add non-scoped style to target `q-tab__content` element (so that the tab's label and close btn will appear in one line)
```
<style lang="scss">
.tab-item {
  .q-tab__content {
      display: flex;
      flex-direction: row;
    }
}
</style>

Reply:
yes done